src/content/docs/it/tutorial/1-setup/3.mdx

```diff
@@ -0,0 +1,71 @@
+---
+type: tutorial
+title: Scrivi la tua prima riga di Astro
+description: |-
+  Tutorial: Costruisci il tuo primo blog Astro —
+  Fai le tue prime modifiche alla home page del tuo progetto tutorial
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+
+
+
+<PreCheck>
+  - Fai le tue prime modifiche al tuo nuovo sito
+</PreCheck>
+
+## Modifica la tua home page
+
+1. Nel tuo editor di codice, naviga nella sezione Explorer a `src/pages/index.astro` e cliccaci sopra per aprire il contenuto del file in una scheda modificabile.
+
+    Il contenuto del tuo file `index.astro` dovrebbero apparire così:
+
+    ```astro title="src/pages/index.astro"
+    ---
+    ---
+
+    <html lang="en">
+      <head>
+        <meta charset="utf-8" />
+        <link rel="icon" type="image/svg+xml" href="/favicon.ico" />
+        <meta name="viewport" content="width=device-width" />
+        <meta name="generator" content={Astro.generator} >
+        <title>Astro</title>
+      </head>
+      <body>
+        <h1>Astro</h1>
+      </body>
+    </html>
+    ```
+
+2. Modifica il contenuto del `<body>` della tua pagina.
+
+    Inserisci nell'editor per modificare il testo del titolo sulla tua pagina e salva le modifiche.
```

Comment:
I think there's something wrong with this phrase, it doesn't feel right. We should rephrase. What's the original phrase? I might be able to suggest an alternative.

Reply:
@ematipico This is the original phrase:"Type in the editor to change the heading text on your page and save the change. "

I believe the most relevant literary translation can be this: "Digita nell'editor per cambiare il testo dell'intestazione sulla tua pagina e salva la modifica.", but I don't really like it that much.

Reply:
Or, rephrasing a little bit the original sentence: "Modifica il file nell'editor, aggiorna l'intestazione della pagina e infine salva il file".

Reply:
+1 for @csarnataro's suggestion!